Sources/AppcuesKit/Presentation/Debugger/PushVerifier.swift

```diff
@@ -64,8 +64,8 @@ internal class PushVerifier {
                 return "Error 9: Foreground presentation handler not implemented"
             case .noForegroundPresentationOption:
                 return "Note: Application is not configured to display foreground notifications"
-            case .serverError:
-                return "Error 500: Server Error"
+            case .serverError(let error):
+                return "Error 500: \(error ?? "Unexpected server error")"
```

Comment:
can we use the status code from the error instead of 500? it could be different things like 400 or 404 in some of these cases I believe.

Reply:
oh good idea!